packages/x-components/src/views/Layout.vue

```diff
@@ -189,11 +229,27 @@
         </template>
 
         <template #main-body>
+          <!-- IdentifierResults -->
+          <div v-if="$x.identifierResults.length">
```

Comment:
Remove this div as the `IdentifierResults` will not render if there are not sku results.

Reply:
Well, that's the point and it makes sense for me. What I am missing?

Reply:
You don't need an `v-if` in the `div` as the component only renders if there are identifier results, I don't know if yo got me.